tests/helpers.py

```diff
@@ -2652,3 +2652,34 @@ def default_volumesnapshotclass(interface_type):
         resource_name=resource_name
     )
     return OCS(**base_snapshot_class.data)
+
+
+@retry(UnexpectedBehaviour, tries=20, delay=10, backoff=1)
+def fetch_used_size(cbp_name, exp_val=None):
+    """
+    Fetch used size in the pool
+
+    Args:
+        exp_val(float): Expected size in GB
+
+    Returns:
+        float: Used size in GB
+    """
+
+    ct_pod = pod.get_ceph_tools_pod()
+    rados_status = ct_pod.exec_ceph_cmd(
+        ceph_cmd=f"rados df -p {cbp_name}"
+    )
+    size_bytes = rados_status['pools'][0]['size_bytes']
+
+    # Convert size to GB
+    used_in_gb = float(
+        format(size_bytes / constants.GB, '.4f')
+    )
+    if exp_val:
+        if not abs(exp_val - used_in_gb) < 1.5:
+            raise UnexpectedBehaviour(
```

Comment:
you can change:
if exp_val==True and abs(exp_val - used_in_gb) > 1.5:
   raise...

Reply:
done